scripts/run_e2e_tests.py

```diff
@@ -592,7 +593,7 @@ def update_flaky_tests_count(sheet, row_index, current_count):
         python_utils.PRINT('** NOTE: Updated sheet for first failing test **')
 
 
-def run_tests(args=None):
+def run_tests(run_count=MAX_RETRY_COUNT, args=None):
```

Comment:
Why is this `MAX_RETRY_COUNT` by default? Since `run_count` is a counter that gets incremented, this default would mean we have already run `MAX_RETRY_COUNT` times

Reply:
The idea was that run_tests should run only once by default.